Source/Core/CesiumTerrainProvider.js

```diff
@@ -167,8 +167,8 @@ define([
                     var deprecationLinkText = 'Check out the new high-resolution Cesium World Terrain';
                     var deprecationLink = 'https://cesium.com/blog/2018/03/01/introducing-cesium-world-terrain/';
                     that._tileCredits = [
-                        new Credit({ text: deprecationText, showOnScreen: true }),
-                        new Credit({ text: deprecationLinkText, link: deprecationLink, showOnScreen: true })
+                        new Credit(deprecationText, true),
```

Comment:
This was only two credits because of non-HTML credit limitations.  You can merge it into a single one now.